src/main/java/com/pnu/ordermanagementapp/service/ClientServiceImpl.java

```diff
@@ -50,13 +56,34 @@ public void create(Client client, Long userId) {
                 .userId(userId)
                 .build();
 
+        if (clientRepository.existsClientByEmailAndUserId(client.getEmail(), userId)) {
+            throw new ServiceException(
+                    "Client with email '" + client.getEmail() + "' already exists (active?: " + client.isActive() +
```

Comment:
Use `String.format()`
No need for `?` after active, it can be confusing
No need in `/n`